src/main/resources/static/js/testrules.js

```diff
@@ -221,14 +190,19 @@ jQuery(document).ready(
               sticky : false,
               theme : 'Notify'
             });
-            var eventsList = JSON.parse(fileContent);
-            ko.cleanNode($("#testEventsDOMObject")[0]);
-            vm.eventsBindingList.removeAll();
-            $('.eventsListDisplay > div:gt(0)').remove();
-            vm.eventsBindingList = ko.observableArray(eventsList);
-            ko.applyBindings(vm, $("#testEventsDOMObject")[0]);
-            closeTooltip();
-            loadTooltip();
+
+            var list = JSON.parse(fileContent);
+
+            if (type == true) {
```

Comment:
Perhaps call this boolean something like 'isRules'

Reply:
Done